doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -366,12 +367,12 @@ the monitor nodes with the `iburst` option and multiple peers:
    into initial synchronization.
 
 Furthermore, it is advisable to synchronize *all* nodes in your cluster against
-internal and external servers, and perhaps even against your monitors. ``NTP``
-servers should run on bare metal; VM virtualized clocks are not suitable for
-steady timekeeping. For more information, visit `https://www.ntp.org
-<https://www.ntp.org>`_.  Your organization might already have quality internal
-``NTP`` servers available.  Sources for ``NTP`` server appliances include the
-following:
+internal and external servers, and perhaps even against your monitors. Run
+``NTP`` servers should on bare metal; VM-virtualized clocks are not suitable
```

Comment:
s/should//
s/;/:/

Reply:
Oh good God. Another superb catch.

and re s/;/:/ : Accepted.